inc/ocf_mngt.h

```diff
@@ -302,6 +313,7 @@ struct ocf_mngt_cache_config {
 static inline void ocf_mngt_cache_config_set_default(
 		struct ocf_mngt_cache_config *cfg)
 {
+	cfg->name_len = sizeof(*cfg->name);
```

Comment:
This is wrong. name_len is supposed to store size of the string. Please remove this from set_default function - whoever sets cache name is responsible for setting this value as well.